_includes/build/best_practices_download_speed.html

```diff
@@ -0,0 +1,25 @@
+<div>
+  <h3>
+    Requesting compressed data
+  </h3>
+
+  <p>
+    Requesting compressed data can help increase your download speed. You can request compressed data files by
+    specifying the `Accept-Encoding: gzip` header in your download requests. After you finish downloading, uncompress
+    (unzip) the file to access your data.
```

Comment:
No need to uncompress; by specifying the header, the user's request agent should uncompress the data due to the returned Content-Encoding header. This just speeds up transport

Reply:
e0b9c03